__tests__/app.test.js

```diff
@@ -478,4 +488,70 @@ describe("GET /api/articles (topic query)", () => {
                 })
             });
     });
+});
+
+describe("GET /api/articles/:article_id (comment_count)", () => {
+    test("responds with a status code: 200 and sends a single article to the client without the comment_count property if article_id is valid", () => {
```

Comment:
I'm confused - this _should_ have a comment_count on it - there is no need to add to the url